Axel (#14)
EDUCE-UBC/MICB405: Rmds/windows.Rmd

-   OpenSSH comes pre-installed if you have Windows 10 version 1803 or newer. If you updated to the latest Windows 10 version, you should already have OpenSSH available to you. If your Windows 10 version is older because you cannot update, follow the [official Microsoft documentation on installing OpenSSH Client](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).

> -   The simplest way to access OpenSSH on Windows is to open either the "Command Prompt" or "PowerShell" applications, discoverable through the search bar. These applications will provide sufficient functionality for all server-based work you will do in MICB405.

Reviewer: Let's decide on only one app—no need to give options. If students know about options themselves, they can make these decisions on their own.

-   Once here, you can proceed to Tutorial 1.

## Additional considerations for Windows {-}

You can make your command line experience more modern and customizeable on Windows with the use of [Windows Terminal](https://docs.microsoft.com/en-us/windows/terminal/get-started). 

> 

Reviewer: I would remove this if you do not want students to use Windows Terminal in general. Let's stick with one option. Don't know if you saw my other [comment](https://github.com/EDUCE-UBC/MICB405/pull/14#issuecomment-915524078). Whatever you decide, then all references should talk about that app, e.g. "command prompt".

Be aware that when you open Command Prompt or Powershell, either directly through the associated app or in a new Windows Terminal tab, that you are using an entirely different local shell [with its own set of commands](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands) unique from Linux. Only after you are connected to the server (using OpenSSH, see above) will you be using Bash. Tread with caution if executing commands other than `ssh` or `scp` from the tutorials while still in Command Prompt or PowerShell (i.e. while not connected to the server).

## Optional: Windows Subsystem for Linux

> You can create a true Linux environment at the command line by setting up [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10). You can then access Bash on a virtualized Linux distribution, either directly through that distribution's Windows app or on Windows Terminal, to practice and run Linux commands on your machine without connecting to the server.

Reviewer: Again, I would not mention Windows terminal if that's not our default.

Author: Sure, sounds good. The good thing about the in-person tutorials is that at least if someone is super confused I can help them

-   OpenSSH comes pre-installed if you have Windows 10 version 1803 or newer. If you updated to the latest Windows 10 version, you should already have OpenSSH available to you. If your Windows 10 version is older because you cannot update, follow the [official Microsoft documentation on installing OpenSSH Client](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse).

> -   The simplest way to access OpenSSH on Windows is to open either the "Command Prompt" or "PowerShell" applications, discoverable through the search bar. These applications will provide sufficient functionality for all server-based work you will do in MICB405.

Reviewer: ```suggestion
-   The simplest way to access OpenSSH on Windows is to open the "Command Prompt" application, discoverable through the search bar. This application will provide sufficient functionality for all server-based work you will do in MICB405.
```

-   Once here, you can proceed to Tutorial 1.

## Additional considerations for Windows {-}

> You can make your command line experience more modern and customizeable on Windows with the use of [Windows Terminal](https://docs.microsoft.com/en-us/windows/terminal/get-started). 
> 

Reviewer: ```suggestion
```

> Be aware that when you open Command Prompt or Powershell, either directly through the associated app or in a new Windows Terminal tab, that you are using an entirely different local shell [with its own set of commands](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands) unique from Linux. Only after you are connected to the server (using OpenSSH, see above) will you be using Bash. Tread with caution if executing commands other than `ssh` or `scp` from the tutorials while still in Command Prompt or PowerShell (i.e. while not connected to the server).

Reviewer: ```suggestion
Be aware that when you open Command Prompt, you are using an entirely different local shell [with its own set of commands](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/windows-commands) unique from Linux. Only after you are connected to the server (using OpenSSH, see above) will you be using Bash. Tread with caution if executing commands other than `ssh` or `scp` from the tutorials while still in Command Prompt (i.e. while not connected to the server).
```

## Optional: Windows Subsystem for Linux

> You can create a true Linux environment at the command line by setting up [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10). You can then access Bash on a virtualized Linux distribution, either directly through that distribution's Windows app or on Windows Terminal, to practice and run Linux commands on your machine without connecting to the server.

Reviewer: ```suggestion
You can create a true Linux environment at the command line by setting up [Windows Subsystem for Linux (WSL)](https://docs.microsoft.com/en-us/windows/wsl/install-win10). You can then access Bash on a virtualized Linux distribution, either directly through that distribution's Windows app or on Command Prompt to practice and run Linux commands on your machine without connecting to the server.
```
